tests/Provider/Doctrine/Fixtures/IncorrectUsageTest.php

```diff
@@ -9,20 +9,22 @@ class IncorrectUsageTest extends TestCase
     public function throwsWhenTryingToDefineTheSameEntityTwice()
     {
         $factory = $this->factory->defineEntity('SpaceShip');
-        $this->assertThrows(function () use ($factory) {
-            $factory->defineEntity('SpaceShip');
-        });
+        
+        $this->expectException(\Exception::class);
+        $this->expectExceptionMessage("Entity 'SpaceShip' already defined in fixture factory");
```

Comment:
Probably makes sense to use the `class` keyword as well, would follow up with a PR.

Reply:
sure